src/pages/managerForm.js

```diff
@@ -1,66 +1,311 @@
+import { useState, useRef } from 'react';
```

Comment:
This file is getting long. Is it possible to break it into smaller components based on sections?

Reply:
Created function to loop over inputs with .map